package.json

```diff
@@ -34,18 +34,27 @@
     "postsetup": "npm run build:dll",
     "clean": "shjs ./internals/scripts/clean.js",
     "clean:all": "npm run analyze:clean && npm run test:clean && npm run build:clean",
-    "generate": "plop --plopfile internals/generators/index.js",
+    "generate": "plop --plopfile internals/generators/index.js && npm run prettier:js",
     "lint": "npm run lint:js && npm run lint:css",
     "lint:eslint": "eslint --ignore-path .gitignore --ignore-pattern internals/scripts",
     "lint:js": "npm run lint:eslint -- . ",
-    "lint:css": "stylelint \"./app/**/*.js\"",
+    "lint:css": "stylelint './app/**/*.js'",
     "lint:staged": "lint-staged",
     "pretest": "npm run test:clean && npm run lint",
+    "prettier:js": "prettier --write './{app, internals}/**/*.js'",
     "test:clean": "rimraf ./coverage",
     "test": "cross-env NODE_ENV=test jest --coverage",
     "test:watch": "cross-env NODE_ENV=test jest --watchAll",
     "coveralls": "cat ./coverage/lcov.info | coveralls"
   },
+  "prettier": {
+    "trailingComma": "es5",
+    "bracketSpacing": true,
+    "semi": true,
+    "singleQuote": true,
+    "jsxBracketSameLine": false,
+    "arrowParens": "always"
+  },
   "lint-staged": {
     "*.js": [
       "lint:eslint",
```

Comment:
We can add prettier to pre-commit hook
```
    "**/*.js": [
      "prettier --write",
      "git add"
    ]
```

Reply:
`git add` will add all unstaged files, if the workplace is dirty, then it will add all the dirty files?

Reply:
Yes it formats all the files and stage everything so not cool.